README-KO.md

```diff
@@ -1156,4 +1157,29 @@ class JavaFriendlyAPI {
 
 `URL`의 [동일성 검사](http://docs.oracle.com/javase/7/docs/api/java/net/URL.html#equals(java.lang.Object)) 는 사실 IP 주소를 알아내기 위해 (블로킹) 네트워크 호출을 합니다.  `URI` 클래스는 필드의 동일성을 확인하고 `URL`의 상위 집합 입니다.
 
+### <a name='misc_well_tested_method'>이미 존재 하는 함수를 다시 개발하는 것 보다는 기존의 잘 테스트 된 함수 사용</a>
```

Comment:
@yunpark93, actually, to me somehow it sounds a little bit awkward but I kind of ended up with writing such. Probably fine as is though.